packages/skill-api/src/core/pages/test.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import {Theme} from '../types'
+
+export type TestErrorTypes = 'default'
+
+export interface TestServerPageParams {
+  theme: Theme
+}
+
+export default function TestPage(props: TestServerPageParams) {
```

Comment:
this is a fairly normal "function component" that get's rendered by preact. I didn't explore the theme stuff deeply, but it looks straight forward.